ui-testing-core/src/main/java/com/avito/android/test/matcher/ToolbarTitleStringMatcher.kt

```diff
@@ -0,0 +1,21 @@
+package com.avito.android.test.matcher
+
+import android.support.test.espresso.matcher.BoundedMatcher
+import android.support.v7.widget.Toolbar
+import android.view.View
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+
+internal class ToolbarTitleStringMatcher(private val textMatcher: Matcher<String>) :
+        BoundedMatcher<View, Toolbar>(Toolbar::class.java) {
```

Comment:
Why don't we use `Matcher<in CharSequence>` in `ToolbarTitleMatcher`? Why special implementation for `String` case?